releasenotes/notes/pylons-int-config-b500a59f78297b88.yaml

```diff
@@ -0,0 +1,4 @@
+---
+other:
+  - |
+    Added distributed tracing setting to integration config for pylons.
```

Comment:
Might be good to be more specific here that we're just adding an environment variable to enable distributed tracing